packages/shared/src/components/Settings.tsx

```diff
@@ -166,21 +173,23 @@ export default function Settings({
           </Switch>
         </div>
       </Section>
-      <Section>
-        <SectionTitle>Companion Widget (beta)</SectionTitle>
-        <div className="flex flex-col items-start pl-1.5 -my-0.5">
-          <Switch
-            inputId="hide-companion-switch"
-            name="hide-companion"
-            className="my-3"
-            checked={!optOutCompanion}
-            onToggle={toggleOptOutCompanion}
-            compact={false}
-          >
-            Enable widget on articles
-          </Switch>
-        </div>
-      </Section>
+      {companionPlacement !== 'off' && (
+        <Section>
+          <SectionTitle>Companion Widget (beta)</SectionTitle>
+          <div className="flex flex-col items-start pl-1.5 -my-0.5">
+            <Switch
+              inputId="hide-companion-switch"
+              name="hide-companion"
+              className="my-3"
+              checked={!optOutCompanion}
+              onToggle={toggleOptOutCompanion}
+              compact={false}
+            >
+              Enable widget on articles
```

Comment:
Is this the final copy and design?
I'm not sure we need a dedicated section for it and the copy doesn't match IMO

Reply:
Small note we need this to turn off the companion else users have no way to disable it. But we can raise the copy comment